### PR TITLE
[PLAYER-5000] Updated react native lib to 0.58.0 version

### PR DESCRIPTION
**Pull request by SergeyBicarte**
React native library updated to 0.58.0 , so we need update dependencies for apps with RN

### CODE REVIEW

File: OoyalaSkinSampleApp/app/build.gradle [hunk 1]
```diff
@@ -94,12 +94,12 @@ dependencies {
...
-    implementation files('libs/OoyalaSkinSDK.aar')
+    //implementation files('libs/OoyalaSDK.aar')
+    //implementation files('libs/OoyalaSkinSDK.aar')
     implementation files('libs/OoyalaSSAISDK.aar')
-    // implementation project(':core')
+     implementation project(':core')
     // implementation project(':ssai')
-    // implementation project(':skin')
+     implementation project(':skin')
     // implementation project(':ima')
```
Comment (AlinaVoronkovaEpam) [inline OoyalaSkinSampleApp/app/build.gradle]: Uncomment `mplementation files('libs/OoyalaSDK.aar')` etc

File: settings.gradle [hunk 2]
```diff
@@ -61,10 +61,10 @@ include ':HeartbeatSampleApp'
...
 // Developer
-//include ':skin'
-//project(':skin').projectDir = new File(settingsDir, '../native-skin/sdk/android/skin/')
-//include ':core'
-//project(':core').projectDir = new File(settingsDir, '../android-sdk/sdk/core/')
+include ':skin'
+project(':skin').projectDir = new File(settingsDir, '../native-skin/sdk/android/skin/')
+include ':core'
+project(':core').projectDir = new File(settingsDir, '../android-sdk/sdk/core/')
 //include ':cast'
```
Comment (AlinaVoronkovaEpam) [inline settings.gradle]: let's  don't change `settings.gradle`

**Review by AlinaVoronkovaEpam — CHANGES_REQUESTED** [review]

File: OoyalaSkinSampleApp/app/build.gradle (hunk 1 above)
Comment (SergeyBicarte) [reply OoyalaSkinSampleApp/app/build.gradle]: Done

File: settings.gradle (hunk 2 above)
Comment (SergeyBicarte) [reply settings.gradle]: Done

**Review by AlinaVoronkovaEpam — APPROVED** [review]

**Review by atopilskii — APPROVED** [review]

**Pull request merged**